theano/tests/test_flake8.py

```diff
@@ -269,6 +269,7 @@
     "misc/tests/test_may_share_memory.py",
     "misc/tests/test_pycuda_theano_simple.py",
     "misc/tests/test_gnumpy_utils.py",
+    "misc/tests/test_pkl_utils.py",
```

Comment:
We don't want to add new as as exception. Can you fix the problem in that file and remove it from here? It should be quick, the file is small.